src/App.js

```diff
@@ -6,18 +6,25 @@ import Categories from './components/Categories';
 import './App.css';
 
 function App() {
+  const addActiveClass = (e) => {
+    const navLinks = document.querySelectorAll('nav ul li a');
+    for (let i = 0; i < navLinks.length; i += 1) {
+      navLinks[i].classList.remove('active');
+    }
+    e.target.classList.add('active');
+  };
+
   return (
     <Router>
       <div>
         <nav>
           <h1>Bookstore CMS</h1>
           <ul>
             <li>
-              <Link to="/">Books</Link>
+              <Link to="/" className="active" onClick={addActiveClass}>BOOKS</Link>
             </li>
-            |
             <li>
-              <Link to="/categories">Categories</Link>
+              <Link to="/categories" onClick={addActiveClass}>CATEGORIES</Link>
             </li>
```

Comment:
- [ ] In the navbar there is an icon missing
![nav](https://user-images.githubusercontent.com/44745855/145434503-f6718c4d-7556-4e2e-8040-b422d5082f9d.png)

Reply:
Check the design to see what I am talking about, make sure your page looks like exactly the design